Sources/Files.swift

```diff
@@ -714,3 +714,25 @@ private extension ObjCBool {
     var boolValue: Bool { return Bool(self) }
 }
 #endif
+
+#if !os(Linux)
+extension FileSystem {
+    /// A reference to the document folder used by this file system. Returns nil on Linux by default.
```

Comment:
You can now remove the comment about Linux, as it won't be available for that platform 🙂